src/app/molecules/message/Message.jsx

```diff
@@ -106,10 +106,13 @@ MessageReply.propTypes = {
   content: PropTypes.string.isRequired,
 };
 
-function MessageContent({ content, isMarkdown, isEdited }) {
+function MessageContent({
+  senderName, content, isMarkdown, isEdited, msgType,
+}) {
```

Comment:
Whoops... didn't realize I "re-formatted" this only only partially.

Let's use this as an opportunity to ask: should the full declaration be kept on a single line, or would you prefer one line per (destructured) param? The more params there are, the easier it gets to merge conflicting changes if it's on multiple lines. (That's why I started changing this but oops forgot to finish.)

Reply:
I use next line when param > 3